docs/docs/spark-procedures.md

```diff
@@ -974,6 +974,38 @@ Collect statistics of the snapshot with id `snap1` of table `my_table` for colum
 CALL catalog_name.system.compute_table_stats(table => 'my_table', snapshot_id => 'snap1', columns => array('col1', 'col2'));
 ```
 
+## Partition Statistics
+
+### `compute_partition_stats`
+
+This procedure computes the stats incrementally from the last snapshot that has partition stats file until the given 
+snapshot (uses current snapshot if not specified) and writes the combined result into a `PartitionStatisticsFile`
+after merging the partition stats. Does a full compute if previous statistics file does not exist. Also registers the 
```

Comment:
Nit: `It also registers the 'PartitionStatisticsFile' to the table metadata.` (complete sentence)